torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1901,8 +1926,10 @@ def hint():
                         # originally.  Otherwise, should only assert that
                         # vr is superset of c_vr
                         if not (c_vr.lower == r.lower and c_vr.upper == r.upper):
+                            constraint = c.render(sources[0])
```

Comment:
???

This defeats the purpose of going lazy. And in any case, isn't c the constraint you want? What you have here is a string